lingua_franca/internal.py

```diff
@@ -456,10 +458,19 @@ def _call_localized_function(func, *args, **kwargs):
             lang_param_index = func_params.index('lang')
             full_lang_code = None
 
+            # Check if we need to add timezone awareness to any datetime object
+            if config.inject_timezones:
+                for k, v in kwargs.items():
+                    if isinstance(v, datetime) and v.tzinfo is None:
+                        kwargs[k] = to_local(v)
+                for idx, v in enumerate(args):
+                    if isinstance(v, datetime) and v.tzinfo is None:
+                        args = args[:idx] + (to_local(v),) + args[idx + 1:]
```

Comment:
as a general principle i agree, but a throw away variable inside of a loop i think its legit to be a single character, just like `i` is the most commonly used in `for` loops. usually when iterating `dict.items()` i see `k, v` used, stands for `key, value`. `idx` also is commonly used when using `enumerate`

if we know the type being returned `v` should always be descriptive, but it can be literally anything in this case

Reply:
I agree with specific respect to `i`/`j` for loops, and `k`/`v` keyvals, only because the latter is canonical Python, and the former is canonical everywhere.

In general I too agree that single-character variables suck, but I almost *expect* `for k, v in`. It's a meaningful unit of code =P

Reply:
Is it really that hard to type the extra letters?  `for key, value`  If I get my way, every Mycroft repository will be running PyLint, which won't take too kindly to the one letter variables.  Another thing, usually more meaningful names can be used.  What does the key represent?  The value?  The code will be easier to read if there is some variable naming that helps with intent.  Sorry, fellas.  Not backing down on this one.

Reply:
would you like to open an issue against the python source code? theres over 100 pages of search results with this pattern

https://github.com/python/cpython/search?q=for

Reply:
at some point you should attempt to collaborate instead of "getting your way"